extension/goplugin/schemas.go

```diff
@@ -433,11 +433,11 @@ func (schema *Schema) delete(filter goext.Filter, requestContext goext.Context,
 		resourceID := mapper.FieldByName(resource, "id").Interface()
 
 		mapFromResource := schema.env.Util().ResourceToMap(resource.Interface())
-		contextTx = contextTx.WithResource(mapFromResource).
+		contextCopy = contextCopy.WithResource(mapFromResource).
```

Comment:
There is a bit different way to copy context among create, update and delete so please do it in the same way.